cmake/onnxruntime_providers.cmake

```diff
@@ -480,6 +484,38 @@ if (onnxruntime_USE_NNAPI)
   set_target_properties(onnxruntime_providers_nnapi PROPERTIES LINKER_LANGUAGE CXX)
 endif()
 
+if (onnxruntime_USE_RKNPU)
+  # set(CMAKE_VERBOSE_MAKEFILE ON)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-variable -Wno-unused-parameter")
+  add_definitions(-DUSE_RKNPU=1)
+  option(DNN_READ_ONNX "" ON)
+  set(DNN_CUSTOM_PROTOC_EXECUTABLE ${ONNX_CUSTOM_PROTOC_EXECUTABLE})
+  option(DNN_CMAKE_INSTALL "" OFF)
+  option(DNN_BUILD_BIN "" OFF)
+  set(RKNPU_DDK_INCLUDE_DIR ${REPO_ROOT}/cmake/external/rknpu_ddk/include)
```

Comment:
as mentioned previously, better to use a build option for rknpu library location.
instead of fetching via git submodule.

Reply:
updated